core.lua

```diff
@@ -74,14 +204,25 @@ function CustomTooltips_DisplayTooltip(button, macroText)
 		local tooltipData = TOOLTIPS[tooltipName:upper()]
 		if tooltipData then
 			heading, body = tooltipData.heading, tooltipData.body
+		else
+			heading, body = SetNotFoundError(tooltipName)
 		end
 	else
-		heading, body = macroText:match("#tooltipdesc ([^\n]+)^([^\n]+)") -- #tooltipdesc heading text^body text
+		-- Do we have an inline tooltip definition?
+		if nil == string.find(macroText, "#tooltipdesc") then return end
```

Comment:
Yea, I come from the lovely land of C++, so comparing things to nil/0/nullptr is common place - I'll change this.